src/Command/ExpireOldBuildsCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PHPCensor\Command;
+
+use PHPCensor\Service\BuildService;
+use PHPCensor\Store\BuildStore;
+use PHPCensor\Store\ProjectStore;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Expire old builds.
+ *
+ * @author David Sloan <dave@d3r.com>
+ */
+class ExpireOldBuildsCommand extends Command
```

Comment:
Could you name command more clearly. I think `deleteOldBuild`/`removeOldBuilds` would be good name.